code/upaya/Gemfile

```diff
@@ -55,7 +55,7 @@ group :development do
   gem 'aws-sdk', '~> 2.0'
   gem 'better_errors'
   gem 'derailed'
-  gem 'binding_of_caller', platforms: [:mri_21]
+  gem 'binding_of_caller', platforms: [:mri_23]
```

Comment:
Not related to this particular gem, but a general note to specify the Ruby version at the top of the Gemfile, below the source: `ruby '2.3.0'`. The `omniauth-saml` gem won't install on certain versions, like 2.1.5.